### PR TITLE
[MIG][12.0] stock_split_picking

### PR DESCRIPTION
**Pull request by andrius-preimantas**

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

You can make readme fragments using new README generator. See https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

File: stock_split_picking/__manifest__.py [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
     'name': 'Split picking',
     'summary': 'Split a picking in two not transferred pickings',
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'category': 'Inventory',
     'author': "Camptocamp, "
               "Tecnativa, "
-              "Odoo Community Association (OCA)",
+              "Odoo Community Association (OCA), ",
+              "Versada "
```
Comment (LoisRForgeFlow) [inline stock_split_picking/__manifest__.py]: Generally a straight-forward migration is a contribution, it does not make you module author.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]

File: stock_split_picking/__manifest__.py (hunk 1 above)
Comment (andrius-preimantas) [reply stock_split_picking/__manifest__.py]: Thanks for clarifying that. Code updated.

**Comment by LoisRForgeFlow** [comment]
@andrius-preimantas Tested on runbot and works fine :+1: 

Could you squash your last 2 commits and check what @rousseldenis has told you?

**Comment by rousseldenis** [comment]
@lreficent

**Review by LoisRForgeFlow — APPROVED** [review]
Great, Thanks!

**Pull request merged**